cardano-git-rev/src/Cardano/Git/Rev.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE MultiWayIf #-}
 {-# LANGUAGE OverloadedStrings #-}
-{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TemplateHaskellQuotes #-}
```

Comment:
Seems that is not supported by `ghc-8.10`.

Reply:
Oh right :weary:, `TemplateHaskellQuotes` was made more useful in GHC 9.0: https://gitlab.haskell.org/ghc/ghc/-/merge_requests/2288

I made `TemplateHaskellQuotes` conditional via CPP, but also happy to just unconditionally use `TemplateHaskell` if that seems less noisy.